src/data/nat/parity.lean

```diff
@@ -138,6 +138,18 @@ begin
   exact @modeq.modeq_mul _ _ 1 _ 1 h₁ h₂
 end
 
+theorem odd_mul : odd (m * n) ↔ odd m ∧ odd n :=
+by simp [not_or_distrib] with parity_simps
+
+theorem even.mul (hm : even m) (n) : even (m * n) :=
+even_mul.mpr $ or.inl hm
+
+theorem mul_even (m) (hn : even n) : even (m * n) :=
+even_mul.mpr $ or.inr hn
```

Comment:
What do you think of the following names?
```suggestion
theorem even.mul_right (hm : even m) (n) : even (m * n) :=
even_mul.mpr $ or.inl hm

theorem even.mul_left (m) (hn : even n) : even (m * n) :=
even_mul.mpr $ or.inr hn
```

Reply:
I like this change!

Reply:
Or perhaps `left` and `right` should be swapped. It would be nice to be consistent, but I doubt any user is going to pick between them without looking at the theorem statement anyway, so isn't a big deal.

Reply:
I'm fine with this; in fact, I actually had this at one point but then switched it to what it is now. I think "left" and "right" should indeed be swapped, though.